tests/login.test.js

```diff
@@ -24,4 +25,46 @@ describe("login endpoint test", () =>{
       expect(res.status).toEqual(401);
     });
   });
+
+  describe("email is not in body", () =>{
+    it("should return 400", async() =>{
+      const res = await request(app)
+        .post("/login")
+        .set("Accept", "application/json")
+        .send({
+          password: "Password"
+        });
+      expect(res.status).toEqual(400);
+    });
+  });
+  describe("password is not in body", () =>{
+    it("should return 400", async() =>{
+      const res = await request(app)
+        .post("/login")
+        .set("Accept", "application/json")
+        .send({
+          email: "txiong@"
+        });
+      expect(res.status).toEqual(400);
+    });
+  });
+  describe("checking for cookie called user", () =>{
+    it("should return a cookie", async() =>{
+      const res = await request(app)
+        .post("/login")
+        .set("Accept", "application/json")
+        .send({
+          email: "txiong@",
+          password: "password"
+        });
+      const cookie = `${res.headers["set-cookie"]}`;
```

Comment:
String interpolation is not necessary.

Change to `const cookie = res.headers["set-cookie"];`